content/api/aggregations/fee-stats/single.mdx

```diff
@@ -38,6 +38,20 @@ server
   });
 ```
 
+```python
+from stellar_sdk import Server
+from stellar_sdk.exceptions import BadResponseError, BadRequestError
+
+server = Server("https://horizon-testnet.stellar.org")
+
+try:
+    server \
+    .fee_stats() \
+    .call()
```

Comment:
Though I appreciate the attempt to keep the line short, this is slightly overkill:
```suggestion
    server.fee_stats().call()
```
is just as readable.